pagopa/pn-frontend
# PN-11527-fe-e-possibile-generare-unapi-key-inserendo-solo-uno-spazio-nel-campo-nome

## Short description
This pull request fixes the minimum character validation for the name field in the NewApiKey form. Previously, the form allowed names with less than 3 characters, which is not desirable. With this fix, the form will now correctly validate that the name has at least 3 characters before allowing submission.

## List of changes proposed in this pull request
- added a minimum of 3 characters to the name field in the validation schema
- updated related tests
- added translations for each language

## How to test
Start PA and try to create an api key with less than 3 character and see if the error appear.
